ApiFirst/ApiFirst/RepositoryList.cs

```diff
@@ -0,0 +1,109 @@
+﻿using DomainModels.Models;
+using DomainServices.Interfaces;
+
+namespace DomainServices.Repositories
+{
+    public class RepositoryList : ICustomerRepository
+    {
+        private readonly List<Customer> List = new();
+
+        public bool Create(Customer model)
+        {
+            model.Id = GetAll().LastOrDefault()?.Id + 1 ?? 0;
```

Comment:
```suggestion
            model.Id = _customers.LastOrDefault()?.Id + 1 ?? 1;
```
Usar a lista renomeada comentada anteriormente e ajustar a lógica de ID, para quando retornar nulo (Quando não existir nenhum usuário para pegar o último ID criado) o ID do usuário seja 1,